File: codeforce/800/Bit++.py
def solve(n, x):
	res = 0

	for i in range(n):
		if (x[i][0] == "+" or x[i][-1] == "+"):
			res += 1
		else:
			res -= 1

n = int(input())
x = []
for i in range(n):
	x.append(input())

print(solve(n, x))

File: codeforce/800/Bit++/Bit++.cpp
#include <iostream>
using namespace std;

void solve() {
    int n, res = 0;
    cin >> n;

    for (int i = 0; i < n; i++) {
        string x;
        cin >> x;

        if (x[0] == '+' || x[2] == '+') {
            res++;
        } else {
            res--;
        }
    }

    cout << res << endl;
}

int main() {
    solve();
    return 0;
}